pkg/spiffe/spiffe.go

```diff
@@ -56,6 +60,11 @@ type bundleDoc struct {
 	RefreshHint int    `json:"spiffe_refresh_hint,omitempty"`
 }
 
+type ClusterTrustDomainInfo struct {
+	ClusterID   string
```

Comment:
Minor question: do we need ClusterID in the struct? It's already the key in the map.

Reply:
obsolete.